python/lsst/the/monster/refcats.py

```diff
@@ -258,6 +311,7 @@ def select_stars(self, catalog, band):
 class GaiaDR3Info(RefcatInfo):
     PATH = "/sdf/data/rubin/shared/the_monster/GAIA_DR3/gaia_dr3"
     NAME = "GaiaDR3"
+    FLAG = 999
```

Comment:
I think the flags should be bits just to make them combinable in the future.  This could be `512` or something.  The others should be `1`, `2`, `4`, `8`, `16`, etc.  None should be 0.  Unless this doesn't have a flag and `0` implies only GaiaDR3 data.

Reply:
OK, changed to bits. I set GaiaDR3 flag to 0 (I don't think it'll ever get used, though).